package.json

```diff
@@ -22,12 +24,16 @@
   "homepage": "https://github.com/fastify/fastify-swagger#readme",
   "devDependencies": {
     "fastify": "^1.1.1",
+    "fs-extra": "^5.0.0",
     "standard": "^11.0.0",
     "swagger-parser": "^4.0.2",
-    "tap": "^11.1.0"
+    "tap": "^11.1.0",
+    "swagger-ui-dist": "3.13.0"
   },
   "dependencies": {
+    "fastify-http-proxy": "^0.2.1",
```

Comment:
I guess fastify-http-proxy should be in `devDependencies`.

Reply:
sorry `fastify-http-proxy` should not be here at all, removed